configs/models/bool_mask_i_jepa.yaml

```
@@ -54,7 +54,7 @@ i_jepa_context_visualization_decoder:
       - [512, 512]
       - [512, 256]
       - [256, 128]
-      - [128, 128]
+      - [128, 64]
```

出力チャネル数を元実装に近いものにしたい
```suggestion
      - [128, 128]
```

---

Decoder部分の計算時間が長くなりすぎるから戻したんだ。Validationの処理まで含めたら案の定間に合わなかった。

---

@zassou65535 リアルタイム性を保てるかということも重要項目で、実際に動かして色々計算時間が間に合うかとか見ると良いと思う。

---

> @zassou65535 リアルタイム性を保てるかということも重要項目で、実際に動かして色々計算時間が間に合うかとか見ると良いと思う。
これって具体的にどういうコマンド叩いて実行時間確かめてる？

---

#94 に書いてる。読んでね。
```sh
python scripts/launch.py experiment=bool_mask_i_jepa_with_videos time_scale=1
```

---

チューンナップまで自分でやるなら変えてOK. それなら文句はないよ。

---

ただ、方針として極力シンプルで軽量なものをスタートラインとすると良いよ